docs/engage/front-end-ui/admin-section/ecosystem-settings.md

```diff
@@ -2,4 +2,104 @@
 id: ecosystem-settings
 title: Ecosystem Settings
 sidebar_position: 3
----
\ No newline at end of file
+---
+
+The **Ecosystem** section in the Engage Admin Panel offers a transparent overview of all wallet addresses, token flows, and allocations within your project’s ecosystem. This section allows administrators to ensure financial clarity and trust by clearly showing how project tokens are held, used, and distributed.
+
+### Overview of the Ecosystem Section?
+
+This section centralizes all wallet-related information and real-time analytics for your token ecosystem. It acts as a **public ledger and admin ledger** that showcases:
+
+- The full list of official project wallets
+- Roles and designations (e.g., Treasury, Rewards, Team, Liquidity)
+- Wallet balances (current and historical)
+- Token distribution breakdowns
+
+---
+
+## Understanding the Allocation graphic
+
+The Allocation Graphic in the Ecosystem section visually represents how your project’s token supply is distributed across key stakeholder categories and functions. It provides a real-time, visual summary of tokenomics in action.
+
+### Allocated Tokens
+
+Allocated tokens refer to the portion of your total token supply that has been assigned to specific purposes or wallets, typically in line with your tokenomics framework.
+
+These include:
+
+- Tokens granted to team, advisors, or investors (whether distributed or still vesting)
+- Tokens set aside for ecosystem programs like staking, grants, or liquidity
+- Tokens locked for governance or future use
+- Any token allocation recorded in the Allocation or Vesting modules
+
+> 🔒 *Allocated does not mean distributed — it simply means the tokens have been earmarked and accounted for, even if they are still locked or vesting.*
+
+
+### Unallocated Tokens
+
+Unallocated tokens are the portion of the supply that remains **unassigned**, meaning they haven’t been:
+
+- Allocated to any wallet
+- Scheduled for vesting
+- Committed to any functional use (liquidity, treasury, etc.)
+
+These are typically held in reserve by the project’s treasury and can be used for:
+
+- Future hires or contributors
+- Strategic partnerships or fundraising
+- DAO-governed grants or proposals
+- Emergency reserves or burns
+
+> 🧠 *Keeping a portion unallocated is common in early or growing ecosystems to maintain flexibility and optionality.*
+---
+
+## Ecosystem Addresses
+
+Each project-owned wallet is displayed in the Ecosystem section with:
+
+- **Wallet address**
+- **Balance of $FACTR or other project tokens**
```

Comment:
I think we should avoid mentioning FACTR since it will always be a different project token for clients

Reply:
Balance of project tokens